Ktisis/Locale/i18n/fr_FR.json

```diff
@@ -105,8 +105,8 @@
 				"enable": "Activer",
 
 				"assignByKeyPress": {
-					"title": "Presser les touches",
```

Comment:
I still don't think it's correct. This text displays in the Input tab and basically shows the user what is the key (or shortcut) that they're currently pressing. `Touches enfoncées` ?